git-archive-all

```diff
@@ -128,7 +163,9 @@ class GitArchiver(object):
                 add_file(file_path, arcname)
         else:
             archive = None
-            archiver = lambda file_path, arcname: self.LOG.info("{} => {}".format(file_path, arcname))
+
+            def archiver(file_path, arcname):
+                self.LOG.info("{0} => {1}".format(file_path, arcname))
```

Comment:
It was for simplicity and to match recent version of git-archive-all. However this breaks with python 2.6.